test/extended/extended_test.go

```diff
@@ -0,0 +1,70 @@
+package extended
+
+import (
+	"flag"
```

Comment:
> This is copying the E2E framework.

Copying or not, why are using a different "flag" library?  Any reason besides cargo-culting?